### PR TITLE
Updating project format

### PR DESCRIPTION
**Pull request by HEskandari**
Use the latest SDK style .csproj format and upgrade to .NET 4.8

### CODE REVIEW

File: src/ServiceInsight.Licensing/InternalVisibleTo.cs [hunk 1]
```diff
@@ -0,0 +1,4 @@
+﻿using System.Runtime.CompilerServices;
+
+[assembly: InternalsVisibleTo("ServiceInsight")]
```
Comment (mikeminutillo) [inline src/ServiceInsight.Licensing/InternalVisibleTo.cs:3]: Why is this necessary? Is this just to prevent us from making the licensing bits public? Is that as important in a product?

File: src/ServiceInsight/MessageFlow/MessageActionPopup.xaml [hunk 2]
```diff
@@ -16,7 +16,7 @@
                 <ResourceDictionary>
                     <SolidColorBrush x:Key="GlyphBrush" Color="#444" />
                     <SolidColorBrush x:Key="DisabledForegroundBrush" Color="#888" />
-                    <SolidColorBrush x:Key="WindowBackgroundBrush" Color="#000" />
+                    <!-- <SolidColorBrush x:Key="WindowBackgroundBrush" Color="#000" /> -->
```
Comment (mikeminutillo) [inline src/ServiceInsight/MessageFlow/MessageActionPopup.xaml:19]: If this is not used, can it be removed?

File: src/ServiceInsight/ServiceInsight.csproj [hunk 3]
```diff
@@ -1,941 +1,94 @@
...
-    <ErrorReport>prompt</ErrorReport>
-    <Prefer32Bit>false</Prefer32Bit>
-    <UseVSHostingProcess>false</UseVSHostingProcess>
-    <CodeAnalysisRuleSet>..\CommonRuleSet.ruleset</CodeAnalysisRuleSet>
-  </PropertyGroup>
-  <PropertyGroup>
+    <Company>NServiceBus Ltd.</Company>
     <ApplicationIcon>ServiceInsight.ico</ApplicationIcon>
+    <UseWpf>true</UseWpf>
+    <DefineConstants>$(DefineConstants);REGISTRYLICENSESOURCE</DefineConstants>
```
Comment (mikeminutillo) [inline src/ServiceInsight/ServiceInsight.csproj:9]: Is this still needed for the main EXE?

File: src/ServiceInsight.Licensing/InternalVisibleTo.cs (hunk 1 above)
Comment (HEskandari) [reply src/ServiceInsight.Licensing/InternalVisibleTo.cs:3]: This is important because we didn't want to make some of our licensing code public. We can't just make part of it public in SI codebase either, since it relies on some other 'internal' bits from the Licensing packages.

File: src/ServiceInsight/ServiceInsight.csproj (hunk 3 above)
Comment (HEskandari) [reply src/ServiceInsight/ServiceInsight.csproj:9]: Probably not. I'll remove and verify.

Comment (HEskandari) [reply src/ServiceInsight/ServiceInsight.csproj:9]: removed

File: src/ServiceInsight/MessageFlow/MessageActionPopup.xaml (hunk 2 above)
Comment (HEskandari) [reply src/ServiceInsight/MessageFlow/MessageActionPopup.xaml:19]: tested and removed
